app/views/patients/_fulfillment.html.erb

```diff
@@ -29,10 +29,11 @@
                           options_for_select(weeks_options, patient.fulfillment.gestation_at_procedure),
                           label: 'Weeks along at procedure',
                           autocomplete: 'off' %>
-            <%= pt.text_field :procedure_date,
+            <%= pt.date_field :procedure_date,
+                                  value: patient.procedure_date_display,
```

Comment:
Might be able to ditch the `value: procedure_date_display` here, since I think the `date_field` helper will automagically handle.